src/nunit.analyzers.tests/DocumentationTests.cs

```diff
@@ -176,9 +176,9 @@ int TableLength()
                 var length = 0;
                 while (startIndex + length < doc.Length)
                 {
-                    if (doc[startIndex + length] == '\n' &&
-                        doc.Length > startIndex + length &&
-                        doc[startIndex + length +1] != '|')
+                    if (doc.Length > startIndex + length &&
```

Comment:
It crashed with index out of bound before, if there was no empty line at the end in 'index.md'

Reply:
I can remember I added an empty line at some point as a quick solution, but I never came back to correct the code. So thanks for correcting this :)